### PR TITLE
[fr] correct a bunch of typos

### PR DESCRIPTION
**Pull request by antograssiot**
I would never expects to find so much :cry:

### CODE REVIEW

File: fr/appendices/3-0-migration-guide.rst [hunk 1]
```diff
@@ -451,7 +451,7 @@ Filter\AssetFilter
 * Les assets des Plugin & theme gérés par AssetFilter ne sont plus lus via
   ``include``, à la place ils sont traités comme de simples fichiers texte. Cela
   règle un certain nombre de problèmes avec les librairies JavaScript comme
-  TinyMCE et les environments avec short_tags activé.
+  TinyMCE et les environnements avec short_tags activé.
```
Comment (markstory) [inline fr/appendices/3-0-migration-guide.rst]: Two nns?

File: fr/appendices/orm-migration.rst [hunk 2]
```diff
@@ -38,7 +38,7 @@ Source de données pour toutes les opérations. Maintenant l'ORM est
 séparé en plus de couches:
 
 * ``Cake\Database\Connection`` - Fournit un moyen de créer et utiliser des
-  connections indépendement de la plateforme. Cette classe permet
+  connections indépendamment de la plateforme. Cette classe permet
```
Comment (markstory) [inline fr/appendices/orm-migration.rst]: two m?

File: fr/appendices/3-0-migration-guide.rst (hunk 1 above)
Comment (antograssiot) [inline fr/appendices/3-0-migration-guide.rst]: yep

File: fr/appendices/orm-migration.rst (hunk 2 above)
Comment (antograssiot) [inline fr/appendices/orm-migration.rst]: and yep

**Comment by cake17** [comment]
:+1: :+1:

**Pull request merged**